lessons/specifics/mnesia.md

```diff
@@ -115,7 +115,7 @@ We define the columns using the atoms `:id`, `:name`, and `:job`. When we execut
 
 ## The Dirty Way
 
-First of all we will look at the dirty way of reading and writing to an Mnesia table. This should generally be avoided as success is not guaranteed, but it should help us learn and become comfortable working with Mnesia. Let's add some entries to our **Person** table.
+First of all we will look at the dirty way of reading and writing to a Mnesia table. This should generally be avoided as success is not guaranteed, but it should help us learn and become comfortable working with Mnesia. Let's add some entries to our **Person** table.
```

Comment:
I don't actually know the proper English rule for this. But, I can see both sides of this. By simple rule, `an` is before a word that starts with a vowel. So in this sense, that is proper. But, by pronunciation aloud it is `am-nē-ZHə` which sounds much akin to `amnesia`. I think since `Mnesia` is not a proper English word, it may make more sense to utilize the phonetically correct rather than the grammatically correct version here, but I am not an expert. It just 'feels' right to me. Heh. Thoughts, anyone?